src/WinRT.Runtime/Projections/INotifyPropertyChanged.net5.cs

```diff
@@ -92,6 +97,7 @@ private static unsafe int Do_Abi_add_PropertyChanged_0(IntPtr thisPtr, IntPtr ha
                 *token = default;
                 try
                 {
+                    global::System.Diagnostics.Debugger.Launch();
```

Comment:
This also was resolved but the leftover `Debugger.Launch()` call is still here? 😅

Reply:
I think I had some lost commits between rebases and different machines, sorry.